src/components/home/Text.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+
+interface ITextProps {
+  value?: string
+}
+
+export class Text extends React.Component<ITextProps, any> {
```

Comment:
I think we don't need this component .. it was a bit confusing .. thinking that we need a special component for rendering text